### DISCUSSION TITLE
Looking for formatting help & curious if tracking numbers are available

### DISCUSSION THREAD

**Discussion by jfgist** [moralmunky/Home-Assistant-Mail-And-Packages#607] [Q&A]
So my dashboard looks like hot garbage. I have just been getting the cards working but I do not have any design skills. I am wondering if there's a good layout example of Mail and Packages. 

I would also like to know if one can access the actual tracking number through HA in addition to having the package counts. 

Thanks

**Comment by firstof9** [comment]
This is how I have my mail view layed out
![image](https://user-images.githubusercontent.com/1105672/146435526-153a0a53-effa-4599-a419-9e6cbf90ce81.png)

and on the right of those cards are the cameras.
